visualizer/src/Project/LabelControls/SegmentControls/ActionButtons/ReplaceButton.js

```diff
@@ -1,21 +0,0 @@
-import React, { useCallback } from 'react';
```

Comment:
Are Swap and Replace functionality implemented elsewhere or are they deprecated by the changes to support overlapping labels?

Reply:
They are deprecated by overlapping labels. Replace is somewhat covered by flood, and swap is only relevant for tracking movies to make cells consistent between frames, so I'll be working on that along with the lineage editing